{{cookiecutter.project_slug}}/Makefile

```diff
@@ -48,12 +48,22 @@ TRAINING_COMMAND?='echo "Replace this placeholder with a training script executi
 
 LOCAL_PORT?=2211
 
+
+# Weights & Biases ingegration settings:
+WANDB_SECRET_FILE?=wandb-token.txt
```

Comment:
Looks like this section is redundant, as it is replaced by the section after GCP.